files/zh-cn/web/css/justify-content/index.md

```diff
@@ -1,11 +1,21 @@
 ---
 title: justify-content
 slug: Web/CSS/justify-content
+page-type: css-property
+browser-compat: css.properties.justify-content
```

Comment:
please remove this part, see: https://github.com/mdn/translated-content/blob/main/docs/zh-cn/translation-guide.md#%E5%85%83%E6%95%B0%E6%8D%AE